src/framework/components/anim/controller.js

```diff
@@ -524,7 +878,8 @@ Object.assign(AnimController.prototype, {
         for (i = 0; i < activeState.animations.length; i++) {
             clip = this._animEvaluator.findClip(activeState.animations[i].name);
             if (!clip) {
-                clip = new AnimClip(activeState.animations[i].animTrack, 0, activeState.speed, true, activeState.loop);
+                var speed = activeState.animations[i].speed ? activeState.animations[i].speed : activeState.speed;
```

Comment:
to confirm, this logic is correct for `activeState.animations[i].speed === 0`?

Reply:
Good catch, i'll use `Number.isFinite` here.